libafl/src/executors/inprocess/inner.rs

```diff
@@ -50,9 +45,7 @@ where
 
 impl<HT, OT, S> Debug for GenericInProcessExecutorInner<HT, OT, S>
 where
-    HT: ExecutorHooksTuple<S>,
```

Comment:
why you deleted these?

Reply:
These are just useless bounds right?

Reply:
no it doesn't build now.

Reply:
ah i think i have one change pending i forgot to push